content/en/docs/contribute/localization.md

```diff
@@ -361,5 +361,3 @@ For more information about working from forks or directly from the repository, s
 ## Upstream contributions
 
 SIG Docs welcomes upstream contributions and corrections to the English source.
-
-
```

Comment:
Can you please undo the removal of these two lines? :)

Reply:
Oh you mean i need to add the two spaces?

Reply:
Added both spaces at the end thanks for the review !